core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -186,6 +187,17 @@ protected double getSpeed(ReaderWay way) {
         return speed;
     }
 
+    @Override
+    public boolean acceptsRelation(OSMTurnRelation relation) {
+        if (relation == null) {
```

Comment:
I added this test because of `OSMReaderTest.testTurnFlagCombination` with the call `reader.analyzeTurnRelation(null)`. Now, to respect your comment and this contraint, I updated the code in `OSMReader.analyzeTurnRelation` to test the `OSMTurnRelation turnRelation` is not null.